gomigrate.go

```diff
@@ -32,11 +32,12 @@ type Migrator struct {
 	MigrationsPath string
 	dbAdapter      Migratable
 	migrations     map[uint64]*Migration
+	schemaName     string
```

Comment:
Could we call this `databaseName` instead? The definition of the word "schema" differs between PG and MySQL.